### ISSUE TITLE
Adding new gambit from testbot crashes Node

### ISSUE THREAD

**Issue by mariusursache** [silentrob/superscript-editor#11]
I tried adding a very simple gambit from the testbot:
- trigger: 'Howdy'
- reply: 'Howdy there'
- topic: random

Node crashes, when restarted gambit is already saved.

Here is the log:

```
/Users/Me/projects/superscript-editor/node_modules/superscript/lib/topics/sort.js:19
    var match  = trig.input.match(/\{weight=(\d+)\}/i);
                     ^
TypeError: Cannot read property 'input' of null
    at Object.exports.sortTriggerSet (/Users/Me/projects/superscript-editor/node_modules/superscript/lib/topics/sort.js:19:22)
    at /Users/Me/projects/superscript-editor/node_modules/superscript/lib/topics/topic.js:70:26
    at /Users/Me/projects/superscript-editor/node_modules/superscript/node_modules/async/lib/async.js:240:13
    at done (/Users/Me/projects/superscript-editor/node_modules/superscript/node_modules/async/lib/async.js:128:19)
    at /Users/Me/projects/superscript-editor/node_modules/superscript/node_modules/async/lib/async.js:25:16
    at /Users/Me/projects/superscript-editor/node_modules/superscript/node_modules/async/lib/async.js:237:17
    at Promise.<anonymous> (/Users/Me/projects/superscript-editor/node_modules/superscript/lib/topics/topic.js:65:9)
    at Promise.<anonymous> (/Users/Me/projects/superscript-editor/node_modules/mongoose/node_modules/mpromise/lib/promise.js:177:8)
    at Promise.emit (events.js:95:17)
    at Promise.emit (/Users/Me/projects/superscript-editor/node_modules/mongoose/node_modules/mpromise/lib/promise.js:84:38)
    at Promise.fulfill (/Users/Me/projects/superscript-editor/node_modules/mongoose/node_modules/mpromise/lib/promise.js:97:20)
    at Object.cb (/Users/Me/projects/superscript-editor/node_modules/mongoose/lib/query.js:1153:30)
    at Object._onImmediate (/Users/Me/projects/superscript-editor/node_modules/mongoose/node_modules/mquery/lib/utils.js:137:16)
    at processImmediate [as _immediateCallback] (timers.js:354:15)

npm ERR! superscript-editor@0.1.4 start: `node app`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the superscript-editor@0.1.4 start script.
npm ERR! This is most likely a problem with the superscript-editor package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app
npm ERR! You can get their info via:
npm ERR!     npm owner ls superscript-editor
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 14.4.0
npm ERR! command "node" "/usr/local/bin/npm" "start"
npm ERR! cwd /Users/Me/projects/superscript-editor
npm ERR! node -v v0.10.35
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
```

**Comment by silentrob** [comment]
Thanks for reporting. I will have a look and push asap.

**Comment by silentrob** [comment]
This is fixed in a7dea4b071c57bef6a3d6525d132ec601ee0e2c9
